applications/ContactStructuralMechanicsApplication/custom_linear_solvers/mixedulm_linear_solver.h

```diff
@@ -2076,6 +2023,13 @@ class MixedULMLinearSolver :
 ///@}
 ///@name Type Definitions
 ///@{
+
+// Here one should use the KRATOS_CREATE_LOCAL_FLAG, but it does not play nice with template parameters
```

Comment:
could you post the error for future reference?

Reply:
Basically is the expansion of the macro which is not designed to template classes